t/conf/extra.conf.in

```diff
@@ -202,7 +202,27 @@ LogLevel        debug
 
 </Directory>
 
-
 RemoteIPHeader          X-Forwarded-For
 RemoteIPProxiesHeader   X-Forwarded-By
 
+<VirtualHost 0.0.0.0:42>
```

Comment:
I think we need to also test the merging of `VirtualHost` config with outer config and maybe with `Directory` config for good measure.